lib/fortnox/api/models/invoice.rb

```diff
@@ -1,142 +1,43 @@
-require "fortnox/api/models/base"
+require "fortnox/api/models/document_base"
 require "fortnox/api/models/edi_information"
-require "fortnox/api/models/email_information"
-require "fortnox/api/models/row"
+require "fortnox/api/models/invoice_row"
```

Comment:
They only share attributes. I'll change it to a mixin instead :) And about the name: I do not think there will be any other `Row`s, but if I use mixin, I think `DocumentRow` will be a lot clearer than `Row`, don't you think?